src/allmydata/scripts/runner.py

```diff
@@ -108,7 +113,7 @@ def postOptions(self):
 
 create_dispatch = {}
 for module in (create_node, stats_gatherer):
-    create_dispatch.update(module.dispatch)
+    create_dispatch.update(module.dispatch)  # type: ignore
```

Comment:
I wondered the same thing. I suspect it's because `(create_node, stats_gatherer)` resolves to `Tuple[ModuleType]` where `ModuleType` is the common interface of the `create_node` and `stats_gatherer` modules and `ModuleType` doesn't generally have a `dispatch` attribute. My guess is one would need to create an interface. to satisfy this check, and that's not simple because it would also require creating a type for the dispatch property as well.